installation/resources/installer-cr-cluster.yaml.tpl

```diff
@@ -9,6 +9,7 @@ metadata:
 spec:
```

Comment:
This file is copied from the master branch, I don't know why it's included here!